codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -645,31 +646,60 @@ addToDependentsIndex dependency dependent = execute sql (dependency :. dependent
     ON CONFLICT DO NOTHING
   |]
 
+-- | Get non-self, user-defined dependents of a dependency.
 getDependentsForDependency :: DB m => Reference.Reference -> m [Reference.Id]
-getDependentsForDependency dependency = query sql dependency where sql = [here|
```

Comment:
Nit: I hate how ormolu breaks this one line into four, that's why there's ormolu_disabled in this file.  https://github.com/tweag/ormolu/issues/696

Reply:
I had a hard time retaining the formatting and also including a second thing in the where-clause. :)